.commitlintrc.yml

```diff
@@ -0,0 +1,82 @@
+# Commitlint configuration.
+# See: https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
+---
+rules:
+  body-leading-blank:
+    - 2
+    - always
+  body-max-line-length:
+    - 2
+    - always
+    - 75
+  footer-leading-blank:
+    - 2
+    - always
+  header-max-length:
+    - 2
+    - always
+    - 75
+  scope-enum:
+    - 2
+    - always
+    - - advisor
+      - analyzer
+      - cli
+      - clients
+      - deps
+      - detekt
+      - docker
+      - downloader
+      - evaluator
+      - examples
+      - github
+      - helper
+      - integrations
+      - model
+      - notifier
+      - reporter
+      - scanner
+      - utils
```

Comment:
Didn't we discuss not to enforce the scopes? It's maybe hard to predict all possible scopes, but we could try with that list.

Reply:
I've decided to drop `scope-enum` as you sometimes might want to specify single file names.